common.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+
+let VM_URL = "https://gw-c7-2b.cloud.victoriametrics.com";
```

Comment:
VM_URL? Let's rename this and also better not to hardcode the default URL.

Reply:
done